Spielregeln.md

```diff
@@ -0,0 +1,98 @@
+Allerdings ist sicherzustellen, dass es möglich ist, das Ziel mit dem Wurfgerät zu Fall zu bringen.
+
+(4) Getrunken wird aus Glasflaschen der 0,5-Liter-Klasse, deren Öffnung DIN EN ISO 12822 oder DIN EN ISO 12821 (handelsübliche Bierflasche mit Kronkorkenverschluss) entspricht.
+Die Getränke müssen Kohlensäure enthalten.
+Es ist explizit erlaubt, Getränke zu konsumieren die keinen Alkohol enthalten bzw. nicht als Bier klassifiziert werden.
+
+(5) Jede spielende Person verfügt über genau ein Getränkebehältnis und trinkt auch nur aus diesem.
+
+## § 4 Spielfeld
+
+(1) In der Mitte des Spielfeldes wird das Ziel aufgestellt.
+
+(2) Von dort ausgehend sind in entgegengesetzte Richtung 6 m (~7 Schritte) abzumessen an denen sich die jeweiligen Grundlinien (auch "Wurflinien") der zwei Teams befinden.
+
+(3) Die Grundlinien der Teams verlaufen parallel.
+
+(4) Das Ziel steht mittig zwischen den beiden Grundlinien.
+
+## § 5 Ablauf
+
+(1) Es gibt immer genau ein Team, welches versucht das Ziel mithilfe des Wurfgeräts zu Fall zu bringen ("werfendes Team") und ein Team, welches darauf reagiert ("aufstellendes Team").
+Nach jedem Wurf wird gewechselt.
+
+(2) Vor Beginn entscheidet ein Münzwurf oder eine Partie Schere-Stein-Papier zwischen zwei Mitgliedern der spielenden Teams über den ersten Wurf.
+Das gewinnende Team entscheidet, welches Team zuerst wirft.
+
+(3) Vor jedem Wurf, muss sichergestellt werden, dass das gegnerische Team bereit ist.
+Dies geschieht durch das Heben einer Hand oder mit einem Zuruf durch das werfende Team, was vom aufstellenden Team durch ein Handheben oder einen zweiten Ruf bestätigt wird.
+Bei Missachtung dieser Regel wird der Wurf annulliert und das Wurfrecht geht an das gegnerische Team über.
+
+(4) Geworfen wird stehend und von oben, d. h. der Ball muss die Wurfhand ungefähr auf Schulterhöhe verlassen.
+Die Grundlinie darf dabei nicht übertreten werden.
+Bei Missachtung dieser Regel wird der Wurf annulliert und das Wurfrecht geht an das gegnerische Team über.
+
+(5) Der Rest des werfenden Teams ist in der Wahl der Körperhaltung frei, darf die Getränkebehältnisse allerdings nicht vor Fall des Zieles berühren.
+
+(6) Sollte das Ziel umfallen, bevor das Wurfgerät die Grundlinie des aufstellenden Teams passiert hat, darf das werfende Team mit dem Trinken beginnen.  
+Getrunken wird so lange, bis das Ziel erfolgreich aufgestellt wurde und sich alle Mitglieder des aufstellenden Teams mitsamt des Wurfgeräts hinter ihrer Grundlinie befinden.
+Der Abschluss dieses Vorgangs ist mit einem deutlich hörbaren "Stop"-Ruf anzuzeigen.  
+Das werfende Team muss unverzüglich alle Trinkaktionen einstellen und die Flaschen zurück auf den Boden führen.
+
+(7) Innerhalb eines Teams wechselt die werfende Person nach jedem Wurf, solange das Team noch über zwei oder mehr Personen verfügt, die ihr Getränk noch nicht geleert haben.
+
+(8) Personen, die ihr Getränk geleert haben, nehmen nicht mehr aktiv am Spiel teil und zählen als ausgeschieden.
```

Comment:
Darf man mit einem Strafschluck abgeben?

Reply:
In der aktuellen Version denk ich ja, es wird nicht definiert, dass man in der eigenen Runde fertig werden muss. Halte ich auch für sinnvoll, wenn die Flasche leer ist, ist sie leer.

Reply:
Spricht vermutlich nichts dagegen. Haltet ihr das für so wichtig, dass das extra spezifiziert werden muss? (Implizit sollte es erlaubt sein)